Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/HolographicCameraNetworkManager.cs

```diff
@@ -8,6 +8,7 @@
 using System.Collections.Generic;
 using System.IO;
 using UnityEngine;
+using UnityEngine.XR.WSA;
 
```

Comment:
does this need to be in UNITY_WSA? If you switch platforms to UNITY_ANDROID or UNITY_IOS, does this still resolve?

Reply:
It does require unity wsa